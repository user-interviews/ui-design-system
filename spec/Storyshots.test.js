import initStoryshots from '@storybook/addon-storyshots';
import './__mocks__/reactDomMock';

initStoryshots();
