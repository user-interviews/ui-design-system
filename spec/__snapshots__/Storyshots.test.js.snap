// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Design System/Buttons/Radio Button Default 1`] = `
<div
  style={
    Object {
      "padding": "1rem",
    }
  }
>
  <input
    className="RadioButton"
    disabled={false}
    id="default"
    name=""
    type="radio"
  />
</div>
`;

exports[`Storyshots Design System/Buttons/Radio Button Disabled 1`] = `
<div
  style={
    Object {
      "padding": "1rem",
    }
  }
>
  <input
    className="RadioButton"
    disabled={true}
    id="disabled"
    name=""
    type="radio"
  />
</div>
`;

exports[`Storyshots Design System/Card all cards 1`] = `
<div
  style={
    Object {
      "padding": "1rem",
    }
  }
>
  <div>
    <div
      className="card"
    >
      <div
        className="card-body"
      >
        <div
          className="card-title h5"
        >
          Test Card
        </div>
        <div>
          Some quick example text to build on the card title and make up the bulk of the card&#39;s content.
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots Design System/Color Palette Blue 1`] = `
<div
  style={
    Object {
      "padding": "1rem",
    }
  }
>
  <div
    style={
      Object {
        "display": "flex",
        "height": "20rem",
        "width": "100%",
      }
    }
  >
    <div
      style={
        Object {
          "backgroundColor": "#EBF6FF",
          "flexBasis": "11%",
          "height": "100%",
        }
      }
    >
      <p
        style={
          Object {
            "backgroundColor": "#FFF",
            "color": "#000",
            "fontSize": "1rem",
            "margin": "1rem",
            "textAlign": "center",
          }
        }
      >
        uxBlue100
      </p>
    </div>
    <div
      style={
        Object {
          "backgroundColor": "#C9E5FC",
          "flexBasis": "11%",
          "height": "100%",
        }
      }
    >
      <p
        style={
          Object {
            "backgroundColor": "#FFF",
            "color": "#000",
            "fontSize": "1rem",
            "margin": "1rem",
            "textAlign": "center",
          }
        }
      >
        uxBlue200
      </p>
    </div>
    <div
      style={
        Object {
          "backgroundColor": "#92CBFC",
          "flexBasis": "11%",
          "height": "100%",
        }
      }
    >
      <p
        style={
          Object {
            "backgroundColor": "#FFF",
            "color": "#000",
            "fontSize": "1rem",
            "margin": "1rem",
            "textAlign": "center",
          }
        }
      >
        uxBlue300
      </p>
    </div>
    <div
      style={
        Object {
          "backgroundColor": "#5AA4E5",
          "flexBasis": "11%",
          "height": "100%",
        }
      }
    >
      <p
        style={
          Object {
            "backgroundColor": "#FFF",
            "color": "#000",
            "fontSize": "1rem",
            "margin": "1rem",
            "textAlign": "center",
          }
        }
      >
        uxBlue400
      </p>
    </div>
    <div
      style={
        Object {
          "backgroundColor": "#337AB7",
          "flexBasis": "11%",
          "height": "100%",
        }
      }
    >
      <p
        style={
          Object {
            "backgroundColor": "#FFF",
            "color": "#000",
            "fontSize": "1rem",
            "margin": "1rem",
            "textAlign": "center",
          }
        }
      >
        uxBlue500
      </p>
    </div>
    <div
      style={
        Object {
          "backgroundColor": "#155B99",
          "flexBasis": "11%",
          "height": "100%",
        }
      }
    >
      <p
        style={
          Object {
            "backgroundColor": "#FFF",
            "color": "#000",
            "fontSize": "1rem",
            "margin": "1rem",
            "textAlign": "center",
          }
        }
      >
        uxBlue600
      </p>
    </div>
    <div
      style={
        Object {
          "backgroundColor": "#0D4473",
          "flexBasis": "11%",
          "height": "100%",
        }
      }
    >
      <p
        style={
          Object {
            "backgroundColor": "#FFF",
            "color": "#000",
            "fontSize": "1rem",
            "margin": "1rem",
            "textAlign": "center",
          }
        }
      >
        uxBlue700
      </p>
    </div>
    <div
      style={
        Object {
          "backgroundColor": "#083054",
          "flexBasis": "11%",
          "height": "100%",
        }
      }
    >
      <p
        style={
          Object {
            "backgroundColor": "#FFF",
            "color": "#000",
            "fontSize": "1rem",
            "margin": "1rem",
            "textAlign": "center",
          }
        }
      >
        uxBlue800
      </p>
    </div>
    <div
      style={
        Object {
          "backgroundColor": "#032847",
          "flexBasis": "11%",
          "height": "100%",
        }
      }
    >
      <p
        style={
          Object {
            "backgroundColor": "#FFF",
            "color": "#000",
            "fontSize": "1rem",
            "margin": "1rem",
            "textAlign": "center",
          }
        }
      >
        uxBlue900
      </p>
    </div>
  </div>
</div>
`;

exports[`Storyshots Design System/Color Palette Gray 1`] = `
<div
  style={
    Object {
      "padding": "1rem",
    }
  }
>
  <div
    style={
      Object {
        "display": "flex",
        "height": "20rem",
        "width": "100%",
      }
    }
  >
    <div
      style={
        Object {
          "backgroundColor": "#F9F9F9",
          "flexBasis": "11%",
          "height": "100%",
        }
      }
    >
      <p
        style={
          Object {
            "backgroundColor": "#FFF",
            "color": "#000",
            "fontSize": "1rem",
            "margin": "1rem",
            "textAlign": "center",
          }
        }
      >
        uxGray100
      </p>
    </div>
    <div
      style={
        Object {
          "backgroundColor": "#F1F1F1",
          "flexBasis": "11%",
          "height": "100%",
        }
      }
    >
      <p
        style={
          Object {
            "backgroundColor": "#FFF",
            "color": "#000",
            "fontSize": "1rem",
            "margin": "1rem",
            "textAlign": "center",
          }
        }
      >
        uxGray200
      </p>
    </div>
    <div
      style={
        Object {
          "backgroundColor": "#E1E1E1",
          "flexBasis": "11%",
          "height": "100%",
        }
      }
    >
      <p
        style={
          Object {
            "backgroundColor": "#FFF",
            "color": "#000",
            "fontSize": "1rem",
            "margin": "1rem",
            "textAlign": "center",
          }
        }
      >
        uxGray300
      </p>
    </div>
    <div
      style={
        Object {
          "backgroundColor": "#D1D1D1",
          "flexBasis": "11%",
          "height": "100%",
        }
      }
    >
      <p
        style={
          Object {
            "backgroundColor": "#FFF",
            "color": "#000",
            "fontSize": "1rem",
            "margin": "1rem",
            "textAlign": "center",
          }
        }
      >
        uxGray400
      </p>
    </div>
    <div
      style={
        Object {
          "backgroundColor": "#A1A1A1",
          "flexBasis": "11%",
          "height": "100%",
        }
      }
    >
      <p
        style={
          Object {
            "backgroundColor": "#FFF",
            "color": "#000",
            "fontSize": "1rem",
            "margin": "1rem",
            "textAlign": "center",
          }
        }
      >
        uxGray500
      </p>
    </div>
    <div
      style={
        Object {
          "backgroundColor": "#818181",
          "flexBasis": "11%",
          "height": "100%",
        }
      }
    >
      <p
        style={
          Object {
            "backgroundColor": "#FFF",
            "color": "#000",
            "fontSize": "1rem",
            "margin": "1rem",
            "textAlign": "center",
          }
        }
      >
        uxGray600
      </p>
    </div>
    <div
      style={
        Object {
          "backgroundColor": "#616161",
          "flexBasis": "11%",
          "height": "100%",
        }
      }
    >
      <p
        style={
          Object {
            "backgroundColor": "#FFF",
            "color": "#000",
            "fontSize": "1rem",
            "margin": "1rem",
            "textAlign": "center",
          }
        }
      >
        uxGray700
      </p>
    </div>
    <div
      style={
        Object {
          "backgroundColor": "#444444",
          "flexBasis": "11%",
          "height": "100%",
        }
      }
    >
      <p
        style={
          Object {
            "backgroundColor": "#FFF",
            "color": "#000",
            "fontSize": "1rem",
            "margin": "1rem",
            "textAlign": "center",
          }
        }
      >
        uxGray800
      </p>
    </div>
    <div
      style={
        Object {
          "backgroundColor": "#222222",
          "flexBasis": "11%",
          "height": "100%",
        }
      }
    >
      <p
        style={
          Object {
            "backgroundColor": "#FFF",
            "color": "#000",
            "fontSize": "1rem",
            "margin": "1rem",
            "textAlign": "center",
          }
        }
      >
        uxGray900
      </p>
    </div>
  </div>
</div>
`;

exports[`Storyshots Design System/Color Palette Green 1`] = `
<div
  style={
    Object {
      "padding": "1rem",
    }
  }
>
  <div
    style={
      Object {
        "display": "flex",
        "height": "20rem",
        "width": "100%",
      }
    }
  >
    <div
      style={
        Object {
          "backgroundColor": "#F3FFF2",
          "flexBasis": "11%",
          "height": "100%",
        }
      }
    >
      <p
        style={
          Object {
            "backgroundColor": "#FFF",
            "color": "#000",
            "fontSize": "1rem",
            "margin": "1rem",
            "textAlign": "center",
          }
        }
      >
        uxGreen100
      </p>
    </div>
    <div
      style={
        Object {
          "backgroundColor": "#E3FDE0",
          "flexBasis": "11%",
          "height": "100%",
        }
      }
    >
      <p
        style={
          Object {
            "backgroundColor": "#FFF",
            "color": "#000",
            "fontSize": "1rem",
            "margin": "1rem",
            "textAlign": "center",
          }
        }
      >
        uxGreen200
      </p>
    </div>
    <div
      style={
        Object {
          "backgroundColor": "#BFFEB8",
          "flexBasis": "11%",
          "height": "100%",
        }
      }
    >
      <p
        style={
          Object {
            "backgroundColor": "#FFF",
            "color": "#000",
            "fontSize": "1rem",
            "margin": "1rem",
            "textAlign": "center",
          }
        }
      >
        uxGreen300
      </p>
    </div>
    <div
      style={
        Object {
          "backgroundColor": "#85DC7A",
          "flexBasis": "11%",
          "height": "100%",
        }
      }
    >
      <p
        style={
          Object {
            "backgroundColor": "#FFF",
            "color": "#000",
            "fontSize": "1rem",
            "margin": "1rem",
            "textAlign": "center",
          }
        }
      >
        uxGreen400
      </p>
    </div>
    <div
      style={
        Object {
          "backgroundColor": "#6DBD63",
          "flexBasis": "11%",
          "height": "100%",
        }
      }
    >
      <p
        style={
          Object {
            "backgroundColor": "#FFF",
            "color": "#000",
            "fontSize": "1rem",
            "margin": "1rem",
            "textAlign": "center",
          }
        }
      >
        uxGreen500
      </p>
    </div>
    <div
      style={
        Object {
          "backgroundColor": "#47A13B",
          "flexBasis": "11%",
          "height": "100%",
        }
      }
    >
      <p
        style={
          Object {
            "backgroundColor": "#FFF",
            "color": "#000",
            "fontSize": "1rem",
            "margin": "1rem",
            "textAlign": "center",
          }
        }
      >
        uxGreen600
      </p>
    </div>
    <div
      style={
        Object {
          "backgroundColor": "#297021",
          "flexBasis": "11%",
          "height": "100%",
        }
      }
    >
      <p
        style={
          Object {
            "backgroundColor": "#FFF",
            "color": "#000",
            "fontSize": "1rem",
            "margin": "1rem",
            "textAlign": "center",
          }
        }
      >
        uxGreen700
      </p>
    </div>
    <div
      style={
        Object {
          "backgroundColor": "#1A5313",
          "flexBasis": "11%",
          "height": "100%",
        }
      }
    >
      <p
        style={
          Object {
            "backgroundColor": "#FFF",
            "color": "#000",
            "fontSize": "1rem",
            "margin": "1rem",
            "textAlign": "center",
          }
        }
      >
        uxGreen800
      </p>
    </div>
    <div
      style={
        Object {
          "backgroundColor": "#0E4207",
          "flexBasis": "11%",
          "height": "100%",
        }
      }
    >
      <p
        style={
          Object {
            "backgroundColor": "#FFF",
            "color": "#000",
            "fontSize": "1rem",
            "margin": "1rem",
            "textAlign": "center",
          }
        }
      >
        uxGreen900
      </p>
    </div>
  </div>
</div>
`;

exports[`Storyshots Design System/Color Palette Orange 1`] = `
<div
  style={
    Object {
      "padding": "1rem",
    }
  }
>
  <div
    style={
      Object {
        "display": "flex",
        "height": "20rem",
        "width": "100%",
      }
    }
  >
    <div
      style={
        Object {
          "backgroundColor": "#FDEAD2",
          "flexBasis": "11%",
          "height": "100%",
        }
      }
    >
      <p
        style={
          Object {
            "backgroundColor": "#FFF",
            "color": "#000",
            "fontSize": "1rem",
            "margin": "1rem",
            "textAlign": "center",
          }
        }
      >
        uxOrange100
      </p>
    </div>
    <div
      style={
        Object {
          "backgroundColor": "#FCDFB9",
          "flexBasis": "11%",
          "height": "100%",
        }
      }
    >
      <p
        style={
          Object {
            "backgroundColor": "#FFF",
            "color": "#000",
            "fontSize": "1rem",
            "margin": "1rem",
            "textAlign": "center",
          }
        }
      >
        uxOrange200
      </p>
    </div>
    <div
      style={
        Object {
          "backgroundColor": "#FAC988",
          "flexBasis": "11%",
          "height": "100%",
        }
      }
    >
      <p
        style={
          Object {
            "backgroundColor": "#FFF",
            "color": "#000",
            "fontSize": "1rem",
            "margin": "1rem",
            "textAlign": "center",
          }
        }
      >
        uxOrange300
      </p>
    </div>
    <div
      style={
        Object {
          "backgroundColor": "#F7B258",
          "flexBasis": "11%",
          "height": "100%",
        }
      }
    >
      <p
        style={
          Object {
            "backgroundColor": "#FFF",
            "color": "#000",
            "fontSize": "1rem",
            "margin": "1rem",
            "textAlign": "center",
          }
        }
      >
        uxOrange400
      </p>
    </div>
    <div
      style={
        Object {
          "backgroundColor": "#F59C27",
          "flexBasis": "11%",
          "height": "100%",
        }
      }
    >
      <p
        style={
          Object {
            "backgroundColor": "#FFF",
            "color": "#000",
            "fontSize": "1rem",
            "margin": "1rem",
            "textAlign": "center",
          }
        }
      >
        uxOrange500
      </p>
    </div>
    <div
      style={
        Object {
          "backgroundColor": "#E18C28",
          "flexBasis": "11%",
          "height": "100%",
        }
      }
    >
      <p
        style={
          Object {
            "backgroundColor": "#FFF",
            "color": "#000",
            "fontSize": "1rem",
            "margin": "1rem",
            "textAlign": "center",
          }
        }
      >
        uxOrange600
      </p>
    </div>
    <div
      style={
        Object {
          "backgroundColor": "#AE6608",
          "flexBasis": "11%",
          "height": "100%",
        }
      }
    >
      <p
        style={
          Object {
            "backgroundColor": "#FFF",
            "color": "#000",
            "fontSize": "1rem",
            "margin": "1rem",
            "textAlign": "center",
          }
        }
      >
        uxOrange700
      </p>
    </div>
    <div
      style={
        Object {
          "backgroundColor": "#7D4A06",
          "flexBasis": "11%",
          "height": "100%",
        }
      }
    >
      <p
        style={
          Object {
            "backgroundColor": "#FFF",
            "color": "#000",
            "fontSize": "1rem",
            "margin": "1rem",
            "textAlign": "center",
          }
        }
      >
        uxOrange800
      </p>
    </div>
    <div
      style={
        Object {
          "backgroundColor": "#653B05",
          "flexBasis": "11%",
          "height": "100%",
        }
      }
    >
      <p
        style={
          Object {
            "backgroundColor": "#FFF",
            "color": "#000",
            "fontSize": "1rem",
            "margin": "1rem",
            "textAlign": "center",
          }
        }
      >
        uxOrange900
      </p>
    </div>
  </div>
</div>
`;

exports[`Storyshots Design System/Color Palette Red 1`] = `
<div
  style={
    Object {
      "padding": "1rem",
    }
  }
>
  <div
    style={
      Object {
        "display": "flex",
        "height": "20rem",
        "width": "100%",
      }
    }
  >
    <div
      style={
        Object {
          "backgroundColor": "#F0D9D9",
          "flexBasis": "11%",
          "height": "100%",
        }
      }
    >
      <p
        style={
          Object {
            "backgroundColor": "#FFF",
            "color": "#000",
            "fontSize": "1rem",
            "margin": "1rem",
            "textAlign": "center",
          }
        }
      >
        uxRed100
      </p>
    </div>
    <div
      style={
        Object {
          "backgroundColor": "#E8C5CB",
          "flexBasis": "11%",
          "height": "100%",
        }
      }
    >
      <p
        style={
          Object {
            "backgroundColor": "#FFF",
            "color": "#000",
            "fontSize": "1rem",
            "margin": "1rem",
            "textAlign": "center",
          }
        }
      >
        uxRed200
      </p>
    </div>
    <div
      style={
        Object {
          "backgroundColor": "#EB7A7A",
          "flexBasis": "11%",
          "height": "100%",
        }
      }
    >
      <p
        style={
          Object {
            "backgroundColor": "#FFF",
            "color": "#000",
            "fontSize": "1rem",
            "margin": "1rem",
            "textAlign": "center",
          }
        }
      >
        uxRed300
      </p>
    </div>
    <div
      style={
        Object {
          "backgroundColor": "#E44E4E",
          "flexBasis": "11%",
          "height": "100%",
        }
      }
    >
      <p
        style={
          Object {
            "backgroundColor": "#FFF",
            "color": "#000",
            "fontSize": "1rem",
            "margin": "1rem",
            "textAlign": "center",
          }
        }
      >
        uxRed400
      </p>
    </div>
    <div
      style={
        Object {
          "backgroundColor": "#FF4E00",
          "flexBasis": "11%",
          "height": "100%",
        }
      }
    >
      <p
        style={
          Object {
            "backgroundColor": "#FFF",
            "color": "#000",
            "fontSize": "1rem",
            "margin": "1rem",
            "textAlign": "center",
          }
        }
      >
        uxRed500
      </p>
    </div>
    <div
      style={
        Object {
          "backgroundColor": "#C71F1F",
          "flexBasis": "11%",
          "height": "100%",
        }
      }
    >
      <p
        style={
          Object {
            "backgroundColor": "#FFF",
            "color": "#000",
            "fontSize": "1rem",
            "margin": "1rem",
            "textAlign": "center",
          }
        }
      >
        uxRed600
      </p>
    </div>
    <div
      style={
        Object {
          "backgroundColor": "#B11B1B",
          "flexBasis": "11%",
          "height": "100%",
        }
      }
    >
      <p
        style={
          Object {
            "backgroundColor": "#FFF",
            "color": "#000",
            "fontSize": "1rem",
            "margin": "1rem",
            "textAlign": "center",
          }
        }
      >
        uxRed700
      </p>
    </div>
    <div
      style={
        Object {
          "backgroundColor": "#9B1818",
          "flexBasis": "11%",
          "height": "100%",
        }
      }
    >
      <p
        style={
          Object {
            "backgroundColor": "#FFF",
            "color": "#000",
            "fontSize": "1rem",
            "margin": "1rem",
            "textAlign": "center",
          }
        }
      >
        uxRed800
      </p>
    </div>
    <div
      style={
        Object {
          "backgroundColor": "#851414",
          "flexBasis": "11%",
          "height": "100%",
        }
      }
    >
      <p
        style={
          Object {
            "backgroundColor": "#FFF",
            "color": "#000",
            "fontSize": "1rem",
            "margin": "1rem",
            "textAlign": "center",
          }
        }
      >
        uxRed900
      </p>
    </div>
  </div>
</div>
`;

exports[`Storyshots Design System/Copy To Clipboard Default 1`] = `
<div
  style={
    Object {
      "padding": "1rem",
    }
  }
>
  <div
    className="CopyToClipboard"
  >
    <span
      className="CopyToClipboard__copy-text"
    >
      Copy me!
    </span>
    <button
      className="btn btn-link btn-link--neutral"
      onClick={[Function]}
      tracking={
        Object {
          "getTrackingData": [Function],
          "trackEvent": [Function],
        }
      }
      type="button"
    >
      <div>
        <svg
          aria-hidden="true"
          className="svg-inline--fa fa-copy fa-w-14 "
          data-icon="copy"
          data-prefix="far"
          focusable="false"
          role="img"
          style={Object {}}
          viewBox="0 0 448 512"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M433.941 65.941l-51.882-51.882A48 48 0 0 0 348.118 0H176c-26.51 0-48 21.49-48 48v48H48c-26.51 0-48 21.49-48 48v320c0 26.51 21.49 48 48 48h224c26.51 0 48-21.49 48-48v-48h80c26.51 0 48-21.49 48-48V99.882a48 48 0 0 0-14.059-33.941zM266 464H54a6 6 0 0 1-6-6V150a6 6 0 0 1 6-6h74v224c0 26.51 21.49 48 48 48h96v42a6 6 0 0 1-6 6zm128-96H182a6 6 0 0 1-6-6V54a6 6 0 0 1 6-6h106v88c0 13.255 10.745 24 24 24h88v202a6 6 0 0 1-6 6zm6-256h-64V48h9.632c1.591 0 3.117.632 4.243 1.757l48.368 48.368a6 6 0 0 1 1.757 4.243V112z"
            fill="currentColor"
            style={Object {}}
          />
        </svg>
      </div>
    </button>
  </div>
</div>
`;

exports[`Storyshots Design System/Flash Flash Message 1`] = `
<div
  style={
    Object {
      "padding": "1rem",
    }
  }
>
  <div
    className="container Flash"
  />
  <div>
    <p>
      Click the button to see a flash message.  Use the knobs to try different types!
    </p>
    <button
      className="btn btn-primary"
      onClick={[Function]}
      type="button"
    >
      Submit
    </button>
  </div>
</div>
`;

exports[`Storyshots Design System/Form Control Label Radio 1`] = `
<div
  style={
    Object {
      "padding": "1rem",
    }
  }
>
  <label
    className="FormControlLabel"
    htmlFor="radio"
  >
    <span
      className="FormControlLabel__control"
    >
      <input
        className="RadioButton"
        disabled={false}
        id="radio"
        name="radio"
        type="radio"
      />
    </span>
    Labeled radio button
  </label>
</div>
`;

exports[`Storyshots Design System/Form Default 1`] = `
<div
  style={
    Object {
      "padding": "1rem",
    }
  }
>
  <form
    className="Form"
    id="form"
    onSubmit={[Function]}
  >
    <div
      className="FormGroup"
      id="first-name"
    >
      <label
        className="InputLabel"
        htmlFor="first-name"
      >
        First name
      </label>
      <input
        className="form-control"
        id="first-name"
        type="text"
      />
    </div>
    <div
      className="FormGroup"
      id="last-name"
    >
      <label
        className="InputLabel"
        htmlFor="last-name"
      >
        Last name
        <span
          className="InputLabel__helper-text"
        >
           (required)
        </span>
      </label>
      <input
        className="form-control"
        id="last-name"
        type="text"
      />
    </div>
    <div
      className="FormGroup"
      id="email"
    >
      <label
        className="InputLabel"
        htmlFor="email"
      >
        Email
      </label>
      <input
        className="form-control"
        id="email"
        type="text"
      />
    </div>
    <div
      className="FormGroup FormGroup--is-invalid"
      id="phone"
    >
      <label
        className="InputLabel"
        htmlFor="phone"
      >
        Phone
        <span
          className="InputLabel__helper-text"
        >
           (required)
        </span>
      </label>
      <input
        className="form-control"
        id="phone"
        type="text"
      />
      <div
        className="FormGroup__invalid-feedback"
      >
        Invalid phone number
      </div>
    </div>
    <div
      className="RadioButtonGroup"
      name="radio-buttons"
    >
      <label
        className="FormControlLabel"
        htmlFor=""
      >
        <span
          className="FormControlLabel__control"
        >
          <input
            className="RadioButton"
            disabled={false}
            id="first"
            name="radio-buttons"
            type="radio"
          />
        </span>
        first choice
      </label>
      <label
        className="FormControlLabel"
        htmlFor=""
      >
        <span
          className="FormControlLabel__control"
        >
          <input
            className="RadioButton"
            disabled={false}
            id="second"
            name="radio-buttons"
            type="radio"
          />
        </span>
        second choice
      </label>
    </div>
    <button
      className="btn btn-primary"
      type="submit"
    >
      Save
    </button>
  </form>
</div>
`;

exports[`Storyshots Design System/Form Group Default 1`] = `
<div
  style={
    Object {
      "padding": "1rem",
    }
  }
>
  <div
    className="FormGroup"
    id="default"
  >
    
    <input
      className="form-control"
      type="text"
    />
  </div>
</div>
`;

exports[`Storyshots Design System/Form Group Required 1`] = `
<div
  style={
    Object {
      "padding": "1rem",
    }
  }
>
  <div
    className="FormGroup"
    id="with-required"
  >
    <label
      className="InputLabel"
      htmlFor=""
    >
      Form Group with label
      <span
        className="InputLabel__helper-text"
      >
         (required)
      </span>
    </label>
    <input
      className="form-control"
      type="text"
    />
  </div>
</div>
`;

exports[`Storyshots Design System/Form Group With Description 1`] = `
<div
  style={
    Object {
      "padding": "1rem",
    }
  }
>
  <div
    className="FormGroup"
    id="with-description"
  >
    <label
      className="InputLabel"
      htmlFor="input"
    >
      Form Group with description
    </label>
    <input
      className="form-control"
      id="input"
      type="text"
    />
    <div
      className="FormGroup__description"
    >
      test description
    </div>
  </div>
</div>
`;

exports[`Storyshots Design System/Form Group With Description And Errors 1`] = `
<div
  style={
    Object {
      "padding": "1rem",
    }
  }
>
  <div
    className="FormGroup FormGroup--is-invalid"
    id="with-description-and-errors"
  >
    <label
      className="InputLabel"
      htmlFor=""
    >
      Form Group with description and errors
    </label>
    <input
      className="form-control"
      type="text"
    />
    <div
      className="FormGroup__invalid-feedback"
    >
      <ol
        className="FormGroup__invalid-feedback__list"
      >
        <li>
          invalid input!
        </li>
      </ol>
    </div>
    <div
      className="FormGroup__description"
    >
      test description
    </div>
  </div>
</div>
`;

exports[`Storyshots Design System/Form Group With Errors 1`] = `
<div
  style={
    Object {
      "padding": "1rem",
    }
  }
>
  <div
    className="FormGroup FormGroup--is-invalid"
    id="with-errors"
  >
    <label
      className="InputLabel"
      htmlFor=""
    >
      Form Group with errors
    </label>
    <input
      className="form-control"
      type="text"
    />
    <div
      className="FormGroup__invalid-feedback"
    >
      <ol
        className="FormGroup__invalid-feedback__list"
      >
        <li>
          invalid input!
        </li>
      </ol>
    </div>
  </div>
</div>
`;

exports[`Storyshots Design System/Form Group With Label 1`] = `
<div
  style={
    Object {
      "padding": "1rem",
    }
  }
>
  <div
    className="FormGroup"
    id="with-label"
  >
    <label
      className="InputLabel"
      htmlFor="input"
    >
      Form Group with label
    </label>
    <input
      className="form-control"
      id="input"
      type="text"
    />
  </div>
</div>
`;

exports[`Storyshots Design System/Form Inline 1`] = `
<div
  style={
    Object {
      "padding": "1rem",
    }
  }
>
  <form
    className="Form Form--inline"
    id="form"
    onSubmit={[Function]}
  >
    <div
      className="FormGroup"
      id="first-name"
    >
      <label
        className="InputLabel"
        htmlFor="first-name"
      >
        First name
      </label>
      <input
        className="form-control"
        id="first-name"
        type="text"
      />
    </div>
    <div
      className="FormGroup"
      id="last-name"
    >
      <label
        className="InputLabel"
        htmlFor="last-name"
      >
        Last name
        <span
          className="InputLabel__helper-text"
        >
           (required)
        </span>
      </label>
      <input
        className="form-control"
        id="last-name"
        type="text"
      />
    </div>
    <div
      className="FormGroup"
      id="description"
    >
      <label
        className="InputLabel"
        htmlFor="description"
      >
        Description
      </label>
      <textarea
        className="form-control"
        id="description"
        type="text"
      />
    </div>
    <button
      className="btn btn-primary"
      type="submit"
    >
      Save
    </button>
  </form>
</div>
`;

exports[`Storyshots Design System/Pill Blue 1`] = `
<div
  style={
    Object {
      "padding": "1rem",
    }
  }
>
  <div
    className="Pill Pill--blue"
  >
    Test pill
  </div>
</div>
`;

exports[`Storyshots Design System/Pill Default 1`] = `
<div
  style={
    Object {
      "padding": "1rem",
    }
  }
>
  <div
    className="Pill"
  >
    Test pill
  </div>
</div>
`;

exports[`Storyshots Design System/Pill Orange 1`] = `
<div
  style={
    Object {
      "padding": "1rem",
    }
  }
>
  <div
    className="Pill Pill--orange"
  >
    Test pill
  </div>
</div>
`;

exports[`Storyshots Design System/Pill Small 1`] = `
<div
  style={
    Object {
      "padding": "1rem",
    }
  }
>
  <div
    className="Pill Pill--sm"
  >
    Test pill
  </div>
</div>
`;

exports[`Storyshots Design System/Popper Dark 1`] = `
<div
  style={
    Object {
      "padding": "1rem",
    }
  }
>
  <div />
  <div
    className="Popper Popper--dark"
    style={
      Object {
        "left": 0,
        "opacity": 0,
        "pointerEvents": "none",
        "position": "absolute",
        "top": 0,
      }
    }
  >
    Dark Popper
  </div>
</div>
`;

exports[`Storyshots Design System/Popper Default 1`] = `
<div
  style={
    Object {
      "padding": "1rem",
    }
  }
>
  <div />
  <div
    className="Popper"
    style={
      Object {
        "left": 0,
        "opacity": 0,
        "pointerEvents": "none",
        "position": "absolute",
        "top": 0,
      }
    }
  >
    Default Popper
  </div>
</div>
`;
